x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/querydsl/query/SingleValueQueryTests.java

```diff
@@ -403,18 +279,5 @@ public List<List<Object>> build(RandomIndexWriter iw) throws IOException {
             }
             return fieldValues;
         }
-
-        @Override
-        public void assertStats(SingleValueQuery.Builder builder, YesNoSometimes subHasTwoPhase) {
-            assertThat(builder.stats().missingField(), equalTo(1));
-            assertThat(builder.stats().numericSingle(), equalTo(0));
-            assertThat(builder.stats().numericMultiNoApprox(), equalTo(0));
-            assertThat(builder.stats().numericMultiApprox(), equalTo(0));
-            assertThat(builder.stats().ordinalsSingle(), equalTo(0));
-            assertThat(builder.stats().ordinalsMultiNoApprox(), equalTo(0));
-            assertThat(builder.stats().ordinalsMultiApprox(), equalTo(0));
-            assertThat(builder.stats().bytesApprox(), equalTo(0));
-            assertThat(builder.stats().bytesNoApprox(), equalTo(0));
```

Comment:
I thought the stats would be useful in debugging slow queries. It's certainly nice to know things like "this rewrote itself to match all in 7/8 segments".

Reply:
But I don't think it's a blocker.